index/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file need to be checked in for something to work? It's the only file in the `index` folder and only seems to cnotain eclipse settings.